src/pages/character-page/character-page.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC, useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { CharacterRequest } from "../../models/request/character-request";
+import "./character-page.scss";
+import { CharacterResult } from "../../models/request/search";
+
+export const CharacterPage: FC = () => {
+    const [characterResult, setCharacterResult] = useState<CharacterResult>();
+
+    const { characterName: encodedCharacterName } = useParams<{
+        characterName: string;
+    }>();
+
+    useEffect(() => {
+        const characterName = decodeURIComponent(encodedCharacterName || "");
+
+        const characterRequest = new CharacterRequest(characterName);
+
+        characterRequest.execute().then((results) => {
+            setCharacterResult(results);
+        });
```

Comment:
should handle error (display "Error" on the page).

Reply:
add an error handler